shared/css/sidebar.css

```diff
@@ -103,106 +103,179 @@ body.sidebar {
-.media-layout > .media-control-buttons > .btn-mute-video:hover:after,
-.media-layout > .media-control-buttons > .btn-mute-video:active:after {
+.media-control-buttons .btn-mute-video:after,
+.media-control-buttons .btn-mute-video:hover:after,
+.media-control-buttons .btn-mute-video:active:after {
   content: url("../img/video.svg");
 }
 
-.media-layout > .media-control-buttons > .btn-mute-video.muted:after {
+.media-control-buttons .btn-mute-video.muted:after {
   content: url("../img/video-muted-darkgrey.svg");
 }
 
-.media-layout > .media-control-buttons > .btn-mute-screen:after {
+.media-control-buttons .btn-mute-screen:after {
   content: url("../img/screen.svg");
 }
-.media-layout > .media-control-buttons > .btn-mute-screen:after,
-.media-layout > .media-control-buttons > .btn-mute-screen:hover:after,
-.media-layout > .media-control-buttons > .btn-mute-screen:active:after {
+.media-control-buttons .btn-mute-screen:after,
+.media-control-buttons .btn-mute-screen:hover:after,
+.media-control-buttons .btn-mute-screen:active:after {
   content: url("../img/screen.svg");
 }
 
-.media-layout > .media-control-buttons > .btn-mute-screen.muted:after {
+.media-control-buttons .btn-mute-screen.muted:after {
   content: url("../img/screen-muted.svg");
 }
 
-.media-layout > .media-control-buttons > .general-support-url {
+.media-control-buttons .general-support-url {
   align-items: center;
   display: flex;
   justify-content: center;
 }
 
 /* Highlight on hover. */
-.media-layout > .media-control-buttons > .general-support-url:hover,
-.media-layout > .media-control-buttons > .btn:not(.btn-hangup):hover {
+.media-control-buttons .general-support-url:hover,
+.media-control-buttons .btn:not(.btn-hangup):hover {
   background-color: rgba(51, 51, 51, 0.1);
 }
 
 /*------------------------------------*\
+    ROOM PRESENCE
+\*------------------------------------*/
+
+.room-active-users {
+  align-items: center;
+  display: flex;
+}
+
+.room-active-users .room-user {
+  width: 30px;
+  height: 30px;
+  line-height: 30px;
+  text-align: center;
+  background-color: #333;
+  color: #fff;
+  border-radius: 50%;
+  position: relative;
+  display: inline-block;
+  margin-left: 10px;
+}
+
+.room-active-users .room-user:hover:after {
+  background: #333;
+  background: rgba(0,0,0,.8);
+  border-radius: 5px;
+  top: 37px;
+  color: #fff;
+  content: attr(data-name);
+  position: absolute;
+  z-index: 98;
+  min-width: 120px;
+  left: 0;
+}
+
+.room-active-users .room-user:hover:before {
+  border: solid;
+  border-color: #333 transparent;
+  border-width: 0 6px 6px;
+  top: 31px;
+  content: "";
+  left: 50%;
+  position: absolute;
+  z-index: 99;
+  transform: translateX(-50%);
+}
+
+
+/*------------------------------------*\
     TEXT CHAT
 \*------------------------------------*/
 
 .text-chat-view {
   flex: 0 0 auto;
   /* Leaves enough room for the streams and the media buttons */
-  height: calc(100% - 228px);
+  height: calc(100% - 188px - 40px);
```

Comment:
anyway, please add a comment with the origin of those sizes to make it easier to keep it sync'ed